observability/logging/logging_release_notes/logging-5-9-release-notes.adoc

```diff
@@ -10,6 +10,8 @@ include::snippets/logging-compatibility-snip.adoc[]
 
 include::snippets/logging-stable-updates-snip.adoc[]
 
+include::modules/logging-release-notes-5-9-10.adoc[leveloffset=+1]
```

Comment:
It might make sense to base this on the 4.16 branch of the documentation, because that one has all the previous release-notes up until 5.9.9

Reply:
Yes, indeed. I've done so in https://github.com/openshift/openshift-docs/pull/86355.